### PR TITLE
fix to the voc broken package 

### LINKED ISSUE #1616: VOC example is broken
## 🐛 Bug description

This PR https://github.com/pytorch/ignite/pull/1605 breakes Pascal VOC example due to bad import:

```python
import torch
from apex import amp
from dataflow.datasets import VOCSegmentationOpencv  # <---- imported before sys.path.insert
from py_config_runner.config_utils import TRAINVAL_CONFIG, assert_config, get_params
from py_config_runner.utils import set_seed
from utils import exp_tracking  # <---- imported before sys.path.insert
from utils.handlers import predictions_gt_images_handler  # <---- imported before sys.path.insert

...

# Adds "code" folder to python path
sys.path.insert(0, Path(__file__).parent.parent.as_posix())
```

## Environment

 - PyTorch Version (e.g., 1.4):
 - Ignite Version (e.g., 0.3.0):
 - OS (e.g., Linux):
 - How you installed Ignite (`conda`, `pip`, source):
 - Python version:
 - Any other relevant information:

### PR DESCRIPTION
**Pull request by sparkingdark**
Fixes #1616 against broken pull req #1605 which broken the import 

tagging @vfdev-5 

closed pr #1636 

Check list:

- [ ] New tests are added (if a new feature is added)
- [ ] New doc strings: description and/or example code are in RST format
- [ ] Documentation is updated (if required)

### CODE REVIEW

File: examples/references/segmentation/pascal_voc2012/code/scripts/training.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 # It should obligatory contain `run(config, **kwargs)` method
-
 import sys
 from collections.abc import Mapping
 from pathlib import Path
 
 import torch
 from apex import amp
+
+sys.path.insert(0, Path(__file__).parent.parent.as_posix())
```
Comment (vfdev-5) [inline examples/references/segmentation/pascal_voc2012/code/scripts/training.py:10]:
```suggestion
# Adds "code" folder to python path
sys.path.insert(0, Path(__file__).parent.parent.as_posix())
```

**Comment by sparkingdark** [comment]
Am i need to add something extra let me know after your testing ?

and one thing is this example available in google colab as well so we can try out easily @vfdev-5

**Review by vfdev-5 — APPROVED** [review]
Thanks for the fix @sparkingdark !

**Comment by sparkingdark** [comment]
Pleasure to work with such great community and learn from everyone.

**Pull request merged**